digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala

```diff
@@ -222,7 +260,7 @@ object TierekisteriDataImporter {
   val tierekisteriDataImporters = Map[String, TierekisteriImporterOperations](
     "litRoad" -> litRoadImporterOperations,
     "roadWidth" -> roadWidthImporterOperations,
-    "trafficSign" -> trafficSignTierekisteriImporter,
+    //"trafficSign" -> trafficSignTierekisteriImporter,
```

Comment:
Commented code

Reply:
could stay commented, The main process continue work, not possible to call